examples/dockerfile/buildx.bzl

```
@@ -14,48 +150,100 @@ def _impl_configure_buildx(rctx):
     compatible_with = "[]"
     builder_name = "builder-docker"
     if has_docker:
-        # TODO(alex): a less hacky way for a repo rule to find this transitive repo label?
-        if repo_utils.platform(rctx) == "darwin_amd64":
-            multitool_label = "@@rules_multitool~~multitool~multitool.macos_x86_64//tools/buildx:macos_x86_64_executable"
-        elif repo_utils.platform(rctx) == "darwin_arm64":
-            multitool_label = "@@rules_multitool~~multitool~multitool.macos_arm64//tools/buildx:macos_arm64_executable"
-        elif repo_utils.platform(rctx) == "linux_amd64":
-            multitool_label = "@@rules_multitool~~multitool~multitool.linux_x86_64//tools/buildx:linux_x86_64_executable"
-        else:
-            fail("platform {} is not known", repo_utils.platform(rctx))
-
-        buildx = rctx.path(Label(multitool_label))
+        buildx = rctx.path(rctx.attr.buildx)
 
         r = rctx.execute([buildx, "ls"])
         if not builder_name in r.stdout:
-            r = rctx.execute([
-                buildx,
-                "create",
-                "--name",
-                builder_name,
-                "--driver",
-                "docker-container",
-                "--use",
-                "--bootstrap",
-            ])
+            r = rctx.execute([buildx, "create", "--name", builder_name, "--driver", "docker-container"])
```

We added `--use --bootstrap` to fix a race condition when this is used in parallel https://github.com/bazel-contrib/rules_oci/pull/775

```suggestion
            r = rctx.execute([buildx, "create", "--name", builder_name, "--driver", "docker-container", "--use", "--bootstrap"])
```

---

right, sorry, we should not have dropped it.